charts/rancher/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
 name: rancher
 description: Install Rancher Server to manage Kubernetes clusters across providers.
-version: 2.7.5
-appVersion: v2.7.5
-kubeVersion: < 1.26.0-0
+version: 2.7.6
+appVersion: v2.7.6
+kubeVersion: < 1.27.0-0
```

Comment:
🤔

Reply:
from @amartin120 in slack:
> yeah.  they actually bumped it for 2.7.5 and I missed it initially.  we ended up fixing the release branch but it never got reflected back in `main`

Reply:
yeah.  they actually bumped it for 2.7.5 and I missed it initially.  we ended up fixing the release branch but it never got reflected back in main.